src/arch/arm/armlib/exceptions/Mybuild

```diff
@@ -2,6 +2,7 @@ package embox.arch.arm.armlib
 
 module exceptions {
 	option number log_level=0
+	option boolean keep_going=true /* Don't stall on undefined exception */
```

Comment:
may be default make false?

Reply:
Before these changes OS will not stop on exception, so I decided to keep the same behavior (for some examples it even works well and application continues to execute normally instead of infinite exception loop).

But I'm not sure what to do here.

Reply:
Ok